graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/kotlin2/Kotlin2TypeLookup.kt

```diff
@@ -30,23 +31,7 @@ import com.squareup.kotlinpoet.LIST
 import com.squareup.kotlinpoet.ParameterizedTypeName.Companion.parameterizedBy
 import com.squareup.kotlinpoet.STRING
 import graphql.Scalars
-import graphql.language.Document
-import graphql.language.EnumTypeDefinition
-import graphql.language.ImplementingTypeDefinition
-import graphql.language.InterfaceTypeDefinition
-import graphql.language.ListType
-import graphql.language.NamedNode
-import graphql.language.Node
-import graphql.language.NodeTraverser
-import graphql.language.NodeVisitorStub
-import graphql.language.NonNullType
-import graphql.language.ObjectTypeDefinition
-import graphql.language.OperationDefinition
-import graphql.language.ScalarTypeDefinition
-import graphql.language.StringValue
-import graphql.language.Type
-import graphql.language.TypeName
-import graphql.language.UnionTypeDefinition
+import graphql.language.*
```

Comment:
I'd prefer to leave the imports expanded

Reply:
I prefer them too. I didn't see any code style rules hence I rolled with IntelliJ defaults and linter 😉